doc/shared/nvectors/NVector_Kokkos.rst

```diff
@@ -187,13 +187,21 @@ class.
 
       Implicit conversion to a :c:type:`N_Vector`.
 
-   .. cpp:function:: N_Vector Convert() override
+   .. cpp:function:: N_Vector get() override
 
       Explicit conversion to a :c:type:`N_Vector`.
+      
+      .. versionadded:: x.y.z
+      
+         Replaces the ``Convert`` method which was deprecated and moved to the ``ConvertibleTo`` class.
```

Comment:
Do the `ConvertibleTo` class need to be mentioned? Users probably won't know what that is.